wgengine/magicsock/magicsock.go

```diff
@@ -1815,9 +1815,11 @@ func debugRingBufferSize(numPeers int) int {
 	}
 	var maxRingBufferSize int
 	if runtime.GOOS == "ios" || runtime.GOOS == "android" {
-		maxRingBufferSize = 1 * 1024 * 1024
+		maxRingBufferSize = 1 << 20
+		// But as of 2024-03-20, we now just disable the ring buffer entirely
+		// on mobile as it hadn't proven useful enough to justify even 1 MB.
 	} else {
-		maxRingBufferSize = 4 * 1024 * 1024
+		maxRingBufferSize = 4 << 20
```

Comment:
😵‍💫

Reply:
Sorry, habit. Go style (at least inside the Go tree) is to use shifts for KB (`<< 10`), MB (`<< 20`), GB (`<< 30`) but only for those values. So you'd write `512 << 10` for 512 KB and not `1 << 19`.

Reply:
Huh, TIL! ✨